### ISSUE TITLE
Make filter persistent

### ISSUE THREAD

**Issue by vasili439** [harbur/kubernetic#279]
Hi! When I try to filter some resources by the name in Search field it becomes always empty when I switch to another resource. In previous version I can set permanent filter in Labels field. Could you please return back permanent filter functionality?

**Comment by dkapanidis** [comment]
Hi,

You can add a permanent filter on the top right search box. [Documentation](https://docs.kubernetic.com/features/crosssections-features/search.html#filter-by-labels) explains it in detail.

**Comment by vasili439** [comment]
No. Since the last version I can choose only from labels list but I can't set "custname" for example. Previous version wasn't affected with this bug

**Comment by dkapanidis** [comment]
Is this really a useful scenario? If "custname" is a name that is delegated between other resources, wouldn't it make sense to exist a label that groups them together so that you can perform cross-section actions?

e.g. if you install a helm app with multiple resources, they should be grouped together under [labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/) which you can then use to filter them out and perform cross-section actions.

I'll reopen it, let's see first if there are more users interested in it.

**Comment by vasili439** [comment]
it useful. we have something like this:
someapp
someapp-backoffice
someapp-backoffice-front
and I want to filter everything what is related to "someapp" or we have several pods which is providing streaming service from different providers and I want to filter everything what is related to "stream"

**Comment by dkapanidis** [comment]
Doing some mining on the issues there are some conflicting expectations:
* #48 expects search to be reset
* this, #181: expects search to be persistent

That's also the reasoning for splitting to two text boxes (global and page specific):
- A user may want to quickly **search** something on a page, but wants to continue working on all resources
- A user may want to **filter** by label (or by text as noted here) and keep that cross-section.

**Comment by vasili439** [comment]
I think both points can be reached if you return the same behavior as it was in previous version: label field could be custom and cross-section and search field reset when you switch between sections
